modules/measure/src/main/java/com/opengamma/strata/measure/cms/CmsMeasureCalculations.java

```diff
@@ -260,7 +260,7 @@ MultiCurrencyAmount currentCash(
 
   //-------------------------------------------------------------------------
   // returns the Ibor index or the CMS leg
-  static IborIndex cmsLegIborIndex(ResolvedCmsTrade trade) {
+  static RateIndex cmsLegIborIndex(ResolvedCmsTrade trade) {
```

Comment:
This method should be renamed,

Reply:
Done. Method used only locally, no impact outside the class.